bedrock-entry.sh

```diff
@@ -96,17 +92,18 @@ case ${VERSION^^} in
     ;;
   *)
     # use the given version exactly
+    if isTrue "$PREVIEW"; then
```

Comment:
There isn’t a boolean variable called PREVIEW. Instead VERSION will be set to “PREVIEW”

Reply:
Hmm... isn't PREVIEW an optional variable that can be set at the environment level before executing this script? I see it above here, even if it's not actively used. I thought that was its original intention... See Lines 6/7.

```shell
: "${TMP_DIR:=/tmp}"
: "${PREVIEW:=false}"
```

Reply:
The switch-case only checks the value of VERSION^^ (uppercased) and compares it to the string literal `PREVIEW`. If a user passes in `PREVIEW` instead of `LATEST`, then they will get the latest PREVIEW version. If the user instead wants a specific version, then they would just pass that version in and it would be handled by the default case. 

If the user passes in $PREVIEW=true for the environment before executing this script, then the only affect it would have is look at the `serverBedrockPreviewLinux` link on the download page for bedrock dedicated server. My understanding is that this key will find the following URL from the download page: https://www.minecraft.net/bedrockdedicatedserver/**bin-linux-preview**/bedrock-server-1.21.50.24.zip

whereas passing in `serverBedrockLinux` to the lookupVersion function will locate the following URL:
https://www.minecraft.net/bedrockdedicatedserver/**bin-linux**/bedrock-server-1.21.31.04.zip

Reply:
Ok, I'll take a closer look at the surrounding logic. 

Did you test all these variations?

Reply:
Ah, I see now what you're saying since it'll now derive from the `serverBedrockPreviewLinux` anchor.